### ISSUE TITLE
Python virtual environment setup not working

### ISSUE THREAD

**Issue by Cmheidelberg** [mintproject/ModelCatalogInsertion#7]
I was following the steps provided in the [caps-cli branch README.md](https://github.com/mintproject/ModelCatalogInsertion/blob/caps-cli/README.md). However, after creating the virtual environment (`python -m venv caps-cli-dev`) the `source caps-cli-dev/bin/activate` does not work. 

I was still able to build the project and test it outside the virtual environment by skipping those steps.

**Comment by dgarijo** [comment]
@maurya-rohit @dhruvp-8 please review

**Comment by dhruvp-8** [comment]
@Cmheidelberg are you running this on Windows bcz the process for creating virtual environment is different for Windows Users

**Comment by dgarijo** [comment]
@dhruvp-8 yes, he's on Windows. Please add in the readme how to proceed then

**Comment by Cmheidelberg** [comment]
I might be a little bit ignorant here, so correct my if im missing something, but why are we using python virtual environment? Is this just to insure consistency across different platforms? 

I think the most convenient way to set this up from a user's perspective would be only running `pip install`. (knocking on wood) With wcm we never had major issues between different machines or OSs.

**Comment by maurya-rohit** [comment]
Actually, when we started working on wcm(talking about MacOS) , we were not able to directly run the wcm by just pip install and it started working when we created the virtual environment. That is why we have the instructions for creating virtual environment on the repo. But again this might be OS specific, if you were able to run wcm without creating any virtual environment then you would not need here as well.
